isis/src/control/objs/BundleUtilities/BundleObservationVector.cpp

```diff
@@ -159,52 +163,18 @@ namespace Isis {
 
       // update image serial number to observation ptr map
       m_imageSerialToObservationMap.insertMulti(bundleImage->serialNumber(), bundleObservation);
-    }
-    return bundleObservation;
-  }
-
-  // addnewIsis vs. CSM?
-  // getCsmObservations()
-  // getIsisObservations()
 
-  // TODO: if we break API anyway just remove this
-  /**
-   * Accesses the number of ISIS position parameters for the
-   * contained BundleObservations.
-   *
-   * @return @b int Returns the total number of position parameters for the BundleObservations
-   */
-  int BundleObservationVector::numberPositionParameters() {
-    int positionParameters = 0;
-
-    // loop over isis observations 
-    for (int i = 0; i < size(); i++) {
-      QSharedPointer<BundleObservation> observation = qSharedPointerDynamicCast<BundleObservation>( at(i) );
-      positionParameters += observation->numberPositionParameters();
-    }
-
-    return positionParameters;
-    // return 0 only CSM observations
-  }
-
-
-  // TODO: if we break API anyway just remove this
-  /**
-   * Accesses the number of ISIS pointing parameters for the
-   * contained BundleObservations.
-   *
-   * @return @b int Returns the total number of pointing parameters for the BundleObservations
-   */
-  int BundleObservationVector::numberPointingParameters() {
-    int pointingParameters = 0;
+      // update instrument ID to observation ptr map
+      // separate held observations out into their own entry
+      if (bundleObservation->numberParameters() == 0) {
+        m_instIdToObservationMap.insertMulti("held", bundleObservation);
```

Comment:
This is how we generate a separate images.csv for held images. This works for the scconfigHeld test, but not for FunctionalTestJigsawHeldList I haven't figured out why yet.